yarn.lock

```diff
@@ -6990,7 +6990,7 @@ gzip-size@5.1.1:
     duplexer "^0.1.1"
     pify "^4.0.1"
 
-handlebars@^4.4.0, handlebars@^4.4.2:
+handlebars@^4.4.0:
```

Comment:
for anyone curious - this comes from https://www.npmjs.com/package/conventional-changelog-writer which comes via lerna, a dev tool. this change means people installing Jest will no longer get handlebars (or uglify)